cvIgnacioCalahorro.html

```diff
@@ -89,30 +83,28 @@ <h2 class='about'>ABOUT ME:</h2>
                     </p>
                     <p class="unis1">
-                        &nbsp;&nbsp;&nbsp;&nbsp;Currently. Graduated expected on May 2022
+                        &nbsp;&nbsp;&nbsp;&nbsp;Currently. Expected to graduate in May 2022
                     </p>
                 </div>
                 <div class="texto-justificado">
                     <p class="unis"> 
-                        Msc in Surveying and Mapping.
+                        MSc in Surveying and Mapping at Aalborg University, DK
                     </p>
                     <p class="unis1">
-                        &nbsp;&nbsp;&nbsp;&nbsp;Aalborg University, DK. Graduated 2020 
+                        &nbsp;&nbsp;&nbsp;&nbsp;Graduated 2020 
                     </p>
                     <p>
-                        &nbsp;&nbsp;Activities and societies: Young Professionals in Denmark is a programme that helps international students in Denmark gain insights into the Danish labour market and working culture.
-                        At the first year. Where I was working on the first semester project analyzing a point cloud,
-                        vectorize and compare measurements made with total station with the ones calculate by a code
-                        developed in Python.
+                        &nbsp;&nbsp;Activities and societies: Young Professionals in Denmark is a programme that helps international students in Denmark gain insights into the Danish labour market and working culture. 
+                        I took part in the programme in the first semester of the studies by working on a project which involved point cloud analysis and total station measurement vectorization and comparison. The whole project was carried out in the Python programming language.
                     </p>
                     <p>
-                        &nbsp;&nbsp;On the second semester I am working on a Indoor Positioning project I am working with the wiffi signal and try to find a cheap way to implement the precision. Also during this semester the main topic of the subject are Statistic and Data Fusion and Positioning Technologies.
+                        &nbsp;&nbsp;In the second semester, I worked on an Indoor Positioning project utilizing Wi-Fi signal to triangulate the position of the object in question TODO: do it cheaply. The main topic of the supportive courses for the project were Statistics and Data Fusion and Positioning Technologies.
                     </p>
                     <p>
-                        &nbsp;&nbsp;Finish the master with the thesis project where I evaluated the quality of dual frequency versus single frequency GNSS mobile phone positioning.
+                        &nbsp;&nbsp;The master's programme was brought to conclusion with the thesis entitled "Single vs Dual-frequency Signal of GNSS on Mobile Phone" where I evaluated the quality of single and dual-frequency signal for mobile phone positioning.
```

Comment:
- "Finish the master with the thesis project" doesn't make sense. Maybe something like "I finished the master...". Anyhoo I rewrote it a little bit.
- Notice that I looked up your thesis to get the name, brought up some memories jaja